actionview/lib/action_view/dependency_tracker.rb

```diff
@@ -59,6 +61,12 @@ class ERBTracker # :nodoc:
         \s*                          # followed by optional spaces
       /x
 
+      # A renderable class e.g. TopicRenderable.new("foo") => "TopicRenderable"
+      RENDERABLE = /
+        (?<renderable>[A-Z][A-Za-z:]*)  # a class name captured as RENDERABLE
+        \.                              # followed by .
```

Comment:
This wouldn't capture something like this:

```erb
<%- my_renderable_instance = MyRenderable.new %>
<%= render my_renderable_instance %>
```

But I guess that's too much to ask for, and I'd happily accept that limitation. But it should be documented.

Reply:
Yeah, I couldn't see any feasible way to get around that.